### PR TITLE
Add codemod to remove unneeded returns from test files

### PR DESCRIPTION
**Pull request by bouk**
@lemonmade this will save a lot of time when translating test files

### CODE REVIEW

File: packages/shopify-codemod/transforms/remove-useless-return-from-test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  function matchLast(matcher) {
+    return (nodes) => nodes.length > 0 && j.match(nodes[nodes.length - 1], matcher);
+  }
+
+  return j(file.source)
+    .find(j.CallExpression, {
+      callee: {
+        type: 'Identifier',
+        name: (name) => FUNCTIONS.indexOf(name) >= 0,
```
Comment (lemonmade) [inline packages/shopify-codemod/transforms/remove-useless-return-from-test.js]: Welp, `includes` can't come soon enough.

File: packages/shopify-codemod/transforms/remove-useless-return-from-test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  const printOptions = options.printOptions || {quote: 'single'};
+
+  function matchLast(matcher) {
+    return (nodes) => nodes.length > 0 && j.match(nodes[nodes.length - 1], matcher);
+  }
+
+  return j(file.source)
+    .find(j.CallExpression, {
+      callee: {
+        type: 'Identifier',
```
Comment (lemonmade) [inline packages/shopify-codemod/transforms/remove-useless-return-from-test.js]: Do you actually need to specify the type?

File: packages/shopify-codemod/transforms/remove-useless-return-from-test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  'test',
+  'xcontext',
+  'xdescribe',
+  'xit',
+  'xspecify',
+];
+
+export default function removeUselessReturnFromTest(file, api, options) {
+  const j = api.jscodeshift;
+  const printOptions = options.printOptions || {quote: 'single'};
```
Comment (lemonmade) [inline packages/shopify-codemod/transforms/remove-useless-return-from-test.js]: Can you copy the parameter destructuring stuff I did for the other transforms?

**Comment by lemonmade** [comment]
Looks really good, reads extremely cleanly. :+1:

File: packages/shopify-codemod/transforms/remove-useless-return-from-test.js (hunk 2 above)
Comment (bouk) [inline packages/shopify-codemod/transforms/remove-useless-return-from-test.js]: yes, the callee can be something other than an identifier

**Pull request merged**